ledger/controllers/surveyor.js

```diff
@@ -36,10 +37,45 @@ const server = async (request, reply, runtime) => {
     surveyor.payload = entry.payload
     surveyor.parentId = entry.parentId
   }
+  return surveyor && addChoices(runtime, surveyor)
+}
 
+async function addChoices (runtime, surveyor) {
+  const payload = surveyor.payload || {}
+  surveyor.payload = payload
+  const adFree = payload.adFree || {}
+  payload.adFree = adFree
+  adFree.choices = await getAdjustedChoices(runtime, 'USD', ['BAT'])
   return surveyor
 }
 
+async function getAdjustedChoices (runtime, base, currencies) {
+  const rates = await runtime.currency.rates(base, currencies)
+  return underscore.mapObject(rates, choicesPrices)
+}
+
+function choicesPrices (ratio) {
+  const basePrice = (new BigNumber(1)).dividedBy(ratio)
+  const table = [
+    [3, 5, 7, 10, 20],
+    [4, 6, 9, 12, 25],
+    [5, 8, 11, 17, 35],
+    [6, 10, 14, 20, 40],
+    [9, 12, 20, 35, 50],
+    [15, 25, 35, 50, 100],
+    [20, 35, 50, 85, 175],
+    [30, 50, 70, 100, 200]
```

Comment:
I think we won't have enough surveyors for some of these larger choices

Reply:
oh, in the daily runner? i guess i just have to up the limit to, say 500 then?

Reply:
is there anything that needs to be adjusted for this?